### DISCUSSION TITLE
Sad day, MacOS 10.12

### DISCUSSION THREAD

**Discussion by BonesGit** [GyulyVGC/sniffnet#215] [General]
Was looking forward to trying this out but unfortunately i'm on an old unsupported MacAir with Sierra 10.12. Apple dropped support and i can't upgrade it any longer.  So when I run Sniffnet I get:

> dyld: Library not loaded: /System/Library/Frameworks/ColorSync.framework/Versions/A/ColorSync
>   Referenced from: /Applications/Sniffnet.app/Contents/MacOS/sniffnet
>   Reason: image not found
> Abort trap: 6

It seems ColorSync framework from Apple is only supported from 10.13+.  Maybe add this to your minimum requirements for Mac :( 
Or any way to get around this dependency?

Thanks all

**Comment by GyulyVGC** [comment]
Hi, I'm sorry to hear that.
I didn't even know about this problem until now.

If you are sure your system cannot support that framework I think there is little to do.

_But there is a possible solution._

The default image is created against the `main`branch which uses the `wgpu` renderer, I can try bundling for you a version built against the `glow-renderer` branch of Sniffnet which uses the `glow` renderer.

I don't know honestly if ColorSync is used by `wgpu`, but if that's the case (and if `glow`doesn't use it), then you should be fine.

**Reply by BonesGit** [reply]
HI. I'm willing to try it out if you don't mind. Here to help :)

**Comment by BonesGit** [comment]
Forgot to post link to ColorSync docs: https://developer.apple.com/documentation/colorsync

**Comment by GyulyVGC** [comment]
Here you are a fresh build using `glow`.
Hope it can solve the problem 🙏
 
[sniffnet.dmg.zip](https://github.com/GyulyVGC/sniffnet/files/11533784/sniffnet.dmg.zip)

**Reply by BonesGit** [reply]
same error

**Reply by GyulyVGC** [reply]
If one of my dependencies (I assume [Iced](https://github.com/iced-rs/iced)) depends on ColorSync and your OS doesn't support it, that's pretty much all I can do 😢

**Reply by BonesGit** [reply]
understood. appreciate the try.
